controllers/apply/views/dashboard.njk

```diff
@@ -0,0 +1,54 @@
+{% extends "layouts/main.njk" %}
+{% from "components/action-card/macro.njk" import actionCard %}
+{% from "components/application-card/macro.njk" import applicationCard with context %}
+{% from "components/user-navigation/macro.njk" import userNavigation with context %}
+
+{% block content %}
+    <main role="main" id="content">
+        <section class="content-box u-inner-wide-only">
+            {{ userNavigation(userNavigationLinks) }}
+
+            <h1 class="t--underline">{{ title }}</h1>
```

Comment:
The title (Latest Application) would not be present if latest application does not exist.

Reply:
This part is intentional (for now). I think there should always be a `h1` on the page, so leaving this as is but plan to review with folks once we have a preview link out.

Reply:
Just confirming, is it the same for the button name being 'Continue' instead of 'Continue with this application' and Last Updated value not having the time ?

Reply:
For now yeah, we're a little tighter on space than was thought so sticking with a short label for now (we can always come back to that), I'm using `.calendar` for the last updated which I think only includes the time if the date is recent.